compiler/src/test/resources/bugs/apps_1177_native_indirect_override_unit.wdl

```diff
@@ -0,0 +1,75 @@
+version 1.0
+
+workflow override {
+
+  # Default
+  call default {}
+
+  # Memory
+  call mem_int {}
+
+  # CPU
+  call cpu_int {}
+}
+
+task default {
+    input {
+        String? mock_input = "asdl"
+    }
+
+    command <<< >>>
+
+    output {
+        String mock_out = ""
+    }
+
+    runtime {
+        dx_app: object {
+                    type: "app",
+                    id: "app-G6G0jX80g1FZX1Z57z3zbG6v",
+                    name: "mock_app_sciprodbuild/0.0.6"
```

Comment:
This app is not accessible to me so perhaps it could be replaced with a public app on staging env, so it's available to whoever runs this (if they want to describe the app or use this script in integration tests as well).

Reply:
I'll try to add apps team to the allowed users.

Reply:
fixed in [6e71262](https://github.com/dnanexus/dxCompiler/pull/339/commits/6e712625cf217365e57d565b513ed83d6731457f), using applet APPS_1177